x-pack/plugins/endpoint/server/routes/alerts/list/lib/pagination.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { get } from 'lodash';
```

Comment:
I'd rather not introduce a dependency of our app on lodash. Could we achieve the same functionality with plain Javascript?

Reply:
@peluja1012 Why not? Plenty of other plugins seem to already depend on it within Kibana. We're just using the `get` functionality to be able to retrieve arbitrarily nested fields from an object by name (so that we can sort by arbitrary params). I'm sure we could write the same utility function, but doesn't make a lot of sense to me, since we'd have to also maintain a bunch of tests to ensure it works correctly.

Reply:
Discussed with @madirey. We should use lodash here.